resources/web/lib/bootstrap/bootstrap.css

```
@@ -9268,4 +9268,4 @@ a.text-dark:hover {
   .table-bordered th {
       border: 1px solid #ddd!important
   }
-}
+}
```

You could add a newline to clean up the diff, but it's up to you. 

```suggestion
}

```